bin/PerlDDS/Response_Monitor.pm

```diff
@@ -0,0 +1,53 @@
+package Response_Monitor;
+
+use strict;
+use Time::HiRes;
+
+sub new {
+  # $delay = seconds between measurement
+  # $drift = allowed msecs of clock drift
+  my ($self, $delay, $drift) = @_;
+
+  # defaults
+  $delay = 1 if !defined $delay;
+  $drift = 500 if !defined $drift;
+
+  if (my $pid = fork) {
+    # parent
+    bless { delay => $delay,
+            drift => $drift,
+            child => $pid }, $self;
+  } else {
+    print "Response monitor running with $delay second delay and $drift msecs of drift\n";
+
+    # child
+    my $last;
+    my $now;
+    my $diff_ms;
+
+    while(1) {
+      $last = Time::HiRes::gettimeofday();
+      Time::HiRes::sleep $delay;
+      $now = Time::HiRes::gettimeofday();
+      $diff_ms = 1000 * ($now - $last - $delay);
+
+      if ( $diff_ms > $drift) {
+        print "Response monitor reports $diff_ms milliseconds drift.\n";
```

Comment:
Do we want this to be a warning so autobuild picks up on this?

Reply:
Good idea. Does autobuild just look for "WARNING:" in the message?

Reply:
Yes, I'm not sure what it's exactly looking for, but that should do it.